__tests__/app.test.js

```diff
@@ -245,6 +245,85 @@ describe('/api/articles', () => {
 	});
 });
 
+describe('/api/articles - Queries', () => {
+	describe('Topic', () => {
+		describe('200: ?topic', () => {
+			it('responds with status 200 and with a list of articles by topic', () => {
+				return request(app)
+					.get('/api/articles?topic=mitch')
+					.expect(200)
+					.then(({ body: { articles } }) => {
+						expect(articles).toHaveLength(11);
+						const correctTopic = articles.every(
+							article => article.topic === 'mitch'
+						);
+						expect(correctTopic).toBe(true);
+					});
+			});
+		});
```

Comment:
We should also test for a topic with no associated articles returning an empty array